modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/useTestflowActions.tsx

```diff
@@ -65,11 +65,12 @@ const useTestflowActions = () => {
 						}
 					)
 				),
-			disabled: true,
+
 			hidden: ({dueStatus}) =>
 				dueStatus?.key === TaskStatuses.IN_ANALYSIS,
```

Comment:
Could check the need for "?", if dueStatus is a required field.


```suggestion
				dueStatus.key === TaskStatuses.IN_ANALYSIS,
```